chainer/testing/function_link.py

```diff
@@ -275,9 +275,13 @@ def do_check():
             grad_outputs = self._generate_grad_outputs(outputs)
             grad_grad_inputs = self._generate_grad_grad_inputs(inputs)
 
+            # Generated `grad_grad_inputs`, the upstream gradients for the
+            # double backward test, may contain `None` for omitted gradients.
+            # These must be propagated to the gradient check.
```

Comment:
Please move this comment after the existing one.

Reply:
Moved.